records_mover/db/bigquery/load_job_config_options.py

```diff
@@ -324,13 +324,14 @@ def add_load_job_csv_config(unhandled_hints: Set[str],
     quiet_remove(unhandled_hints, 'datetimeformat')
 
     if hints.datetimeformattz in ['YYYY-MM-DD HH:MI:SSOF',
+                                  'YYYY-MM-DD HH24:MI:SSOF',
                                   'YYYY-MM-DD HH:MI:SS']:
         pass
     else:
         cant_handle_hint(fail_if_cant_handle_hint, 'datetimeformattz', hints)
     quiet_remove(unhandled_hints, 'datetimeformattz')
 
-    if hints.timeonlyformat == 'HH24:MI:SS':
+    if hints.timeonlyformat in ['HH24:MI:SS', 'HH:MI:SS']:
```

Comment:
This fixes issues found during testing!

Reply:
Specifically, the Redshift spec for date/time formats defines HH as equivalent to HH24 - so there's no reason now to handle it similarly to the HH24 cases.  These were reproduced with test cases defined in datetime_cases.py in tests/component below.